app/mailers/request_mailer.rb

```diff
@@ -229,10 +229,12 @@ def comment_on_alert_plural(info_request, count, earliest_unalerted_comment)
   # actual original mail sent by the authority in the admin interface (so
   # can check that attachment decoding failures are problems in the message,
   # not in our code). ]
-  def self.receive(raw_email)
```

Comment:
The commit message should describe the outline of what's happening in this commit.

* What do the 4 sources do?
* How do the feature flags affect what happens?